Header header
float32[] x_center
float32[] y_center
int32 num_items
bool status